### PR TITLE
Slim 496 sm management

### PR DESCRIPTION
**Pull request by eduarddrenth**

### CODE REVIEW

**Comment by CLAAdmin** [comment]
[CLA assistant check](http://52.16.237.165:80/OSGP/Integration-Tests?pullRequest=30) <br/>You should sign our Contributor License Agreement in order to get your pull request merged.<br/>

File: integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/smartmeteringmanagement/FindCommunicationEventsReads.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+package com.alliander.osgp.platform.cucumber.smartmeteringmanagement;
+
+import com.alliander.osgp.adapter.ws.schema.smartmetering.management.EventLogCategory;
+
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+
+public class FindCommunicationEventsReads extends AbstractFindEventsReads {
```
Comment (robindenadel) [inline integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/smartmeteringmanagement/FindCommunicationEventsReads.java]: Nothing is prepared or tested in this class

File: integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/smartmeteringmanagement/FindFraudEventsReads.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    @When("^the find fraud events request is received$")
+    @Override
+    public void theFindEventsRequestIsReceived() throws Throwable {
+        super.theFindEventsRequestIsReceived();
+    }
+
+    @Then("^fraud events should be returned$")
+    @Override
+    public void eventsShouldBeReturned() throws Throwable {
+        super.eventsShouldBeReturned();
```
Comment (robindenadel) [inline integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/smartmeteringmanagement/FindFraudEventsReads.java]: add assertions

File: integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/smartmeteringmanagement/FindMbusEventsReads.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    @When("^the find mbus events request is received$")
+    @Override
+    public void theFindEventsRequestIsReceived() throws Throwable {
+        super.theFindEventsRequestIsReceived();
+    }
+
+    @Then("^mbus events should be returned$")
+    @Override
+    public void eventsShouldBeReturned() throws Throwable {
+        super.eventsShouldBeReturned();
```
Comment (robindenadel) [inline integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/smartmeteringmanagement/FindMbusEventsReads.java]: add assertions

File: integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/smartmeteringmanagement/FindStandardEventsReads.java [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    @When("^the find standard events request is received$")
+    @Override
+    public void theFindEventsRequestIsReceived() throws Throwable {
+        super.theFindEventsRequestIsReceived();
+    }
+
+    @Then("^standard events should be returned$")
+    @Override
+    public void eventsShouldBeReturned() throws Throwable {
+        super.eventsShouldBeReturned();
```
Comment (robindenadel) [inline integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/smartmeteringmanagement/FindStandardEventsReads.java]: add assertions

File: integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/RunCukesTest.java [hunk 5]
```diff
@@ -17,8 +17,7 @@
 import cucumber.api.junit.Cucumber;
 
 @RunWith(Cucumber.class)
-@CucumberOptions(features = "src/test/resources/", tags = {}, plugin = { "pretty",
-        "html:target/output/Cucumber-report", "html:target/output/Cucumber-html-report.html" }, snippets = SnippetType.CAMELCASE)
+@CucumberOptions(features = "src/test/resources/", tags = {}, plugin = { "pretty", "html:target/output/Cucumber-report" }, snippets = SnippetType.CAMELCASE)
```
Comment (robindenadel) [inline integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/RunCukesTest.java]: add "html:target/output/Cucumber-html-report.html" again

File: integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/smartmeteringmanagement/FindCommunicationEventsReads.java (hunk 1 above)
Comment (eduarddrenth) [inline integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/smartmeteringmanagement/FindCommunicationEventsReads.java]: yes there is, under the hood through soap-ui: schema validation and not soap-fault.

File: integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/smartmeteringmanagement/FindFraudEventsReads.java (hunk 2 above)
Comment (eduarddrenth) [inline integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/smartmeteringmanagement/FindFraudEventsReads.java]: The assertions that were there (regex comparisons) are completely covered by schema validation.
I will try if there is anything usefull to assert additional to this.

File: integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/RunCukesTest.java (hunk 5 above)
Comment (eduarddrenth) [inline integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/RunCukesTest.java]: ok, why is that?

Comment (robindenadel) [inline integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/RunCukesTest.java]: the story you're working on should change behaviour like this

**Pull request merged**